R/get_pids.R

```diff
@@ -67,7 +67,7 @@ get_pids <- function(pkg_doi, data_url = NULL){
                      metadata = pids$identifier[pids$formatType == "METADATA"],
                      data = data_id %||% pids$identifier[pids$formatType == "DATA"])
     
-    if(!is.null(data_id) & !any(stringr::str_detect(pids$identifier[pids$formatType == "DATA"], data_id))){
+    if(!is.null(data_id) & !any(stringr::str_detect(pids$identifier[pids$formatType == "DATA"], pid_list$data))){
```

Comment:
I reworked this also.  I think I _did_ mean data_id, but I'll add a bit more context.